### PR TITLE
DEVDOCS-5241: [update] Add tokenized card

### PR DESCRIPTION
**Pull request by bc-traciporter**
# [DEVDOCS-5241]
{Ticket number or summary of work}

## What changed?
Provide a bulleted list in the present tense
* 

## Anything else?
Add related PRs, salient notes, ticket numbers, etc.

ping {names}

[DEVDOCS-5241]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-5241?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: reference/payments/process_payments.yml [hunk 1]
```diff
@@ -80,6 +80,8 @@ paths:
                         - $ref: '#/components/schemas/StoredPayPalAccount'
                         - $ref: '#/components/schemas/GiftCertificate'
                         - $ref: '#/components/schemas/StoreCredit'
+                        - $ref: '#/components/schemas/CreditCard'
```
Comment (rabinshr) [inline reference/payments/process_payments.yml:83]: I don't believe we need `CreditCard`, it is already covered by `Card` component.
Both the examples seem to be related to tokenized cards, one with the iin and expiration date, and other without them.
I am going to loop in @nkostohryz or @bc-alextop to fill in the correct names here.

Comment (alstarodubtcev) [reply reference/payments/process_payments.yml:83]: @rabinshr @bc-traciporter Yes, seems that both examples is TokenizedCard, one with required only field(token) and other with all optional fields.

Comment (alstarodubtcev) [reply reference/payments/process_payments.yml:83]: @bc-traciporter yes

File: reference/payments/process_payments.yml [hunk 2]
```diff
@@ -487,6 +499,44 @@ components:
...
+          type: string
+          description: Type to classify this payment instrument (required)
+          enum:
+            - tokenized_card
+        token:
+          description: Identifier representing this stored PayPal account (required)
+          type: string
+          minLength: 64
+          maxLength: 64
+        inn:
```
Comment (rabinshr) [inline reference/payments/process_payments.yml:525]:
```suggestion
        iin:
```

File: reference/payments/process_payments.yml [hunk 3]
```diff
@@ -487,6 +499,44 @@ components:
...
+          expiration_month: 12
+          expiration_year: 2030
+      properties:
+        type:
+          type: string
+          description: Type to classify this payment instrument (required)
+          enum:
+            - tokenized_card
+        token:
+          description: Identifier representing this stored PayPal account (required)
```
Comment (rabinshr) [inline reference/payments/process_payments.yml:521]:
```suggestion
          description: Identifier representing the tokenized card (required)
```

File: reference/payments/process_payments.yml [hunk 4]
```diff
@@ -487,6 +499,44 @@ components:
...
           type: store_credit
+    TokenizedCard:
+      title: Tokenized Card
+      type: object
+      x-internal: false
+      x-examples:
+        example-1:
+          type: tokenized_card
+          token: K7KW-M9GX-6PQ3
+          inn: 4111111
```
Comment (rabinshr) [inline reference/payments/process_payments.yml:510]:
```suggestion
          iin: 4111111
```

**Review by rabinshr — COMMENTED** [review]
Some typos but otherwise looks good to me 👍

**Review by rabinshr — APPROVED** [review]
LGTM. Thanks @bc-traciporter

File: reference/payments/process_payments.yml [hunk 5]
```diff
@@ -487,6 +499,44 @@ components:
...
+          expiration_month: 12
+          expiration_year: 2030
+      properties:
+        type:
+          type: string
+          description: Type to classify this payment instrument (required)
+          enum:
+            - tokenized_card
+        token:
+          description: Identifier representing the tokenized card (required)
```
Comment (bc-andreadao) [inline reference/payments/process_payments.yml:521]:
```suggestion
          description: Identifier representing the tokenized card (required).
```

File: reference/payments/process_payments.yml [hunk 6]
```diff
@@ -487,6 +499,44 @@ components:
...
+          type: string
+          minLength: 64
+          maxLength: 64
+        iin:
+          type: string
+          description: Issuer identification number
```
Comment (bc-andreadao) [inline reference/payments/process_payments.yml:527]:
```suggestion
          description: Issuer identification number.
```

File: reference/payments/process_payments.yml [hunk 7]
```diff
@@ -487,6 +499,44 @@ components:
...
+        last_four_digits:
+          type: string
+          description: Last four numbers of this card
```
Comment (bc-andreadao) [inline reference/payments/process_payments.yml:530]:
```suggestion
          description: Last four numbers of this card.
```

File: reference/payments/process_payments.yml [hunk 8]
```diff
@@ -487,6 +499,44 @@ components:
...
+        expiration_month:
+          type: string
+          description: Expiry month of this card
```
Comment (bc-andreadao) [inline reference/payments/process_payments.yml:533]:
```suggestion
          description: Expiry month of this card.
```

File: reference/payments/process_payments.yml [hunk 9]
```diff
@@ -487,6 +499,44 @@ components:
...
+        expiration_year:
+          type: string
+          description: Expiry year of this card
```
Comment (bc-andreadao) [inline reference/payments/process_payments.yml:536]:
```suggestion
          description: Expiry year of this card.
```

**Review by bc-andreadao — DISMISSED** [review]
I left minor copyedits

**Pull request merged**